types/jsonwebtoken/index.d.ts

```diff
@@ -177,7 +178,7 @@ export function sign(
  * [options] - Options for the verification
  * returns - The decoded token.
  */
-export function verify(token: string, secretOrPublicKey: Secret, options?: VerifyOptions): object | string;
+export function verify(token: string, secretOrPublicKey: Secret, options?: VerifyOptions): { [key: string]: any } | string;
```

Comment:
```suggestion
export function verify(token: string, secretOrPublicKey: Secret, options?: VerifyOptions): { [key: string]: unknown } | string;
```

Same reason as above.